src/core/View.js

```diff
@@ -506,9 +506,9 @@ export default class View extends Component {
     }
 
     if (this.cubeAxes.setVisibility(showCubeAxes)) {
-      this.cubeAxes
-        .getActors()
-        .forEach(({ setVisibility }) => setVisibility(showCubeAxes));
+      Array.from(this.cubeAxes.getActors()).forEach(({ setVisibility }) =>
+        setVisibility(showCubeAxes)
```

Comment:
Why `Array.from`?

Reply:
I was hitting an error where getActors() was returning null and forEach works on arrays. This change ensures that it is always an array.